src/main/java/core/basesyntax/FruitShopApp.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+import core.basesyntax.dao.FruitDao;
+import core.basesyntax.dao.FruitDaoImpl;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.ReportBuilder;
+import core.basesyntax.service.ReportBuilderImpl;
+import core.basesyntax.service.TransactionParser;
+import core.basesyntax.service.TransactionParserImpl;
+import java.util.List;
+
+public class FruitShopApp {
+    private TransactionParser parser = new TransactionParserImpl();
+    private ReportBuilder reportBuilder = new ReportBuilderImpl();
+    private FruitDao dao = new FruitDaoImpl();
+
+    public static void main(String[] args) {
+        FruitShopApp app = new FruitShopApp();
+        List<FruitTransaction> fruitTransactions = app.parser.getTransactions("transactions.csv");
```

Comment:
![image](https://user-images.githubusercontent.com/81781479/186446076-509438e3-440a-414d-9d62-a5696ad65b2b.png)
you should create FileReader, FileWriter, TransactionParser, FruitShopService, ReportCreator and implementations, OperationHandler and handler for each operation
Implement pattern strategy